### PR TITLE
Openpifpaf TensorRT support, with fixes on tracking

### PR DESCRIPTION
**Pull request by undefined-references**
* Added Openpifpaf GPU TensorRT support for Openpifpaf version 0.12a4                                                                                     
 * Added tracked objects id to visualization and tracked objects to objects list                                                                           
 
 Since master is broken (#88), we made a branch from commit 01b6cb294cc053aec658e5ded0d15ac8e68f4683 which is the latest working commit on master, and applied our changes into that branch. You can make a branch from the same commit and merge this PR into that branch. Whenever master is fine, you can merge it to that branch and merge back to master after resolving conflicts.

### CODE REVIEW

**Comment by undefined-references** [comment]
only x86-gpu-tensorrt-openpifpaf.Dockerfile is tested, other dockerfiles on other platforms are not tested.

File: README.md [hunk 1]
```diff
@@ -188,6 +188,20 @@ Notice: you must have Docker >= 19.03 to run the container with `--gpus` flag.
...
+
+Note that you should have [Nvidia Docker Toolkit](https://github.com/NVIDIA/nvidia-docker) to run the app with GPU support
+```bash
+
+
+# 1) Build Docker image (This step is optional, you can skip it if you want to pull the container from neuralet dockerhub)
+docker build -f x86-gpu-tensorrt-openpifpaf.Dockerfile -t "neuralet/smart-social-distancing:latest-x86_64_gpu_tensorrt" .
+
+# 2) Run Docker container:
+Notice: you must have Docker >= 19.03 to run the container with `--gpus` flag.
```
Comment (mats-claassen) [inline README.md:201]: I think this comment could be added outside of the bash block either before or after. Or left here as a comment (with # prefixed)

Comment (undefined-references) [reply README.md:201]: It was a copy from previous block, but you are right.

Comment (undefined-references) [reply README.md]: fixed.

File: libs/utils/visualization_utils.py [hunk 2]
```diff
@@ -261,6 +261,7 @@ def visualize_boxes_and_labels_on_image_array(
         classes,
         scores,
         colors,
+        tracked_id,
```
Comment (mrn-mln) [inline libs/utils/visualization_utils.py:264]: Could you add it to docstring.

**Comment by mats-claassen** [comment]
Thanks @undefined-references for all the work. I rebased it and adapted it to use supervisor and merged it to master in #91. Will close this PR. If you want to address any of the remaining comments just open a new PR. Otherwise we will do it

**Pull request closed without merging**